Lib/fontTools/ttLib/tables/T_S_I__0.py

```diff
@@ -3,7 +3,7 @@
 from . import DefaultTable
 import struct
 
-tsi0Format = '>HHl'
+tsi0Format = '>HHL'
```

Comment:
Unfortunately no, the VTT private tables' spec is not publicly available. I could only obtain some of this information indirectly via informal conversations.
I'll try to add more comments to make clear what the purpose of these tables is.